server/graphql/v1/queries.js

```diff
@@ -675,7 +675,7 @@ const queries = {
         query.where.status = args.status;
       }
       if (args.category) {
-        query.where.category = { [Op.iLike]: args.category };
+        query.where[Op.and] = [sequelize.literal(`tags[1] ILIKE ${sequelize.escape(args.category)}`)];
```

Comment:
**[:sparkles: **Enhancement**]** Would be better to search among all tags (not sure about the syntax below)

```suggestion
        query.where.tags = { [Op.contains]: [args.category] };
```

Reply:
Good idea, the operator is `var ILIKE ANY(tags)`.